src/ServiceControl/Recoverability/Grouping/Retries/RetryOperationCompletedHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace ServiceControl.Recoverability
+{
+    using NServiceBus;
+    using Raven.Client;
+    using ServiceBus.Management.Infrastructure.Settings;
+    using System.Linq;
+
+    public class RetryOperationCompletedHandler : IHandleMessages<RetryOperationCompleted>
+    {
+        private readonly Settings settings;
+        public IDocumentStore Store { get; set; }
+        public RetryOperationManager RetryOperationManager { get; set; }
+
+        public void Handle(RetryOperationCompleted message)
+        {
+            var completedOperation = new CompletedRetryOperation
+            {
+                RequestId = message.RequestId,
+                RetryType = message.RetryType,
+                CompletionDate = message.CompletionDate
+            };
```

Comment:
SC restart that cut the retry operation short for example. From the end user, that would not be a full success.